tests/format/typescript/module/namespace_nested.ts

```diff
@@ -0,0 +1,15 @@
+declare namespace abc1.def {}
+
+export declare namespace abc2.def {}
+
+export namespace abc3.def {}
+
+namespace abc4.def {}
+
+declare namespace abc5.def.ghi {}
+
+export declare namespace abc2.def.ghi {}
+
+export namespace abc3.def.ghi {}
+
+namespace abc4.def.ghi {}
```

Comment:
Can you also replace `namespace` with `module` keyword, and test them?

Reply:
done [4c60892](https://github.com/prettier/prettier/pull/15249/commits/4c6089266ee192cd8324180c7e72cc32ab75f3ca)